### PR TITLE
Add Service Worker support in Safari 11.1

### LINKED ISSUE #1776: Update service worker support for iOS and macOS
In Safari 11.1 service worker support for iOS and macOS was added but it is still listed as in dev on mdn 

https://developer.apple.com/library/content/releasenotes/General/WhatsNewInSafari/Articles/Safari_11_1.html

### PR DESCRIPTION
**Pull request by abdonrd**
Fixes #1776.

### CODE REVIEW

File: api/Navigator.json [hunk 1]
```diff
@@ -281,21 +281,35 @@
...
+                  }
+                ]
+              },
+              {
+                "version_added": "17"
+              }
+            ],
             "edge_mobile": {
-              "version_added": "17"
+              "version_added": null
```
Comment (ExE-Boss) [inline api/Navigator.json]: Edge mobile tends to be the same as Edge desktop.

/cc @erikadoyle

File: api/Navigator.json [hunk 2]
```diff
@@ -281,21 +281,35 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/serviceWorker",
           "support": {
+            "webview_android": {
+              "version_added": "40"
+            },
```
Comment (ExE-Boss) [inline api/Navigator.json]: `webview_android` should be at the end of the `support` block (issues #1662 and #398)

/cc @jpmedley

File: api/Navigator.json [hunk 3]
```diff
@@ -309,9 +323,6 @@
             "opera_android": {
               "version_added": "27"
             },
-            "qq_android": {
-              "version_added": null
-            },
```
Comment (ExE-Boss) [inline api/Navigator.json]: I generally prefer to keep these

File: api/Navigator.json [hunk 4]
```diff
@@ -320,15 +331,6 @@
             },
             "samsunginternet_android": {
               "version_added": null
-            },
-            "uc_android": {
-              "version_added": null
-            },
-            "uc_chinese_android": {
-              "version_added": null
```
Comment (ExE-Boss) [inline api/Navigator.json]: I generally prefer to keep these

File: api/Navigator.json [hunk 5]
```diff
@@ -320,15 +331,6 @@
...
-            },
-            "webview_android": {
-              "version_added": "40"
             }
```
Comment (ExE-Boss) [inline api/Navigator.json]: `webview_android` should be at the end of the `support` block (issues #1662 and #398)

/cc @jpmedley

File: api/Navigator.json (hunk 2 above)
Comment (abdonrd) [reply api/Navigator.json]: Thanks for the review, done!

File: api/Navigator.json (hunk 5 above)
Comment (abdonrd) [reply api/Navigator.json]: Thanks for the review, done!

File: api/Navigator.json (hunk 1 above)
Comment (abdonrd) [reply api/Navigator.json]: Then should we update it everywhere?

File: api/Navigator.json (hunk 3 above)
Comment (abdonrd) [reply api/Navigator.json]: Then should we update it everywhere? I copied from the SW and these did not appear anywhere.

File: api/Navigator.json (hunk 4 above)
Comment (abdonrd) [reply api/Navigator.json]: Then should we update it everywhere? I copied from the SW and these did not appear anywhere.

File: api/Navigator.json (hunk 3 above)
Comment (ExE-Boss) [reply api/Navigator.json]: Well, the default value for keys that aren’t present is `null`, so adding `qq_android`, `uc_android` and `uc_chinese_android` to the rest isn’t necessary, but removing them from here can potentially lead to merge conflicts, should a PR which adds data for them get merged before this.

File: api/Navigator.json (hunk 1 above)
Comment (ExE-Boss) [reply api/Navigator.json]: I’m currently waiting for **Erika Doyle** for confirmation, but I expect version `17` to be correct.

Comment (erikadoyle) [reply api/Navigator.json]: The flag was removed and SW works by default in EdgeHTML 17 builds, however *null* is fine for current mobile status. Although Fetch/SW is plumbed into the codebase, mobile is not explicitly supported/tested at this time.

**Comment by Elchi3** [comment]
@abdonrd can you rebase this?

**Comment by abdonrd** [comment]
@Elchi3 I just merged `master`!

**Comment by abdonrd** [comment]
Yay! 🎉

**Pull request merged**